<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="JARVIS" name="com.project.test.AutoPilot" tests="7" failures="1" timestamp="16 Jul 2020 13:25:34 GMT" time="12.664" errors="0">
  <testcase name="gotoRegistrationPage" time="11.842" classname="com.project.test.AutoPilot"/>
  <testcase name="fillRegistrationForm" time="0.801" classname="com.project.test.AutoPilot">
    <failure type="java.lang.AssertionError" message="Stopping test.">
      <![CDATA[java.lang.AssertionError: Stopping test.
at com.project.test.AutoPilot.stopTest(AutoPilot.java:241)
at com.project.test.AutoPilot.fillRegistrationForm(AutoPilot.java:106)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- fillRegistrationForm -->
  <testcase name="signOutAfterRegistration" time="0.012" classname="com.project.test.AutoPilot"/>
  <testcase name="loginWithNewAccount" time="0.003" classname="com.project.test.AutoPilot"/>
  <testcase name="searchForProduct" time="0.002" classname="com.project.test.AutoPilot"/>
  <testcase name="selectProduct" time="0.002" classname="com.project.test.AutoPilot"/>
  <testcase name="addToCart" time="0.002" classname="com.project.test.AutoPilot"/>
</testsuite> <!-- com.project.test.AutoPilot -->
